integration/README.md

```diff
@@ -0,0 +1,704 @@
+# Running Snap in various environments
```

Comment:
Let's remove "integration" directory and move everything up, so starting with README.md, run, src, etc.
Let's not merge this PR with master branch, but rather push "integration" branch with this changes.
After migration we would also need "run/gce" directory with provision-snap.sh (former provision-kubesnap.sh)